### DISCUSSION TITLE
Doesn't detect Chrome on 20.0.2

### DISCUSSION THREAD

**Discussion by newviews** [will-stone/browserosaurus#615] [General]
Hi,

Browsersaurus has just stopping finding Chrome as a supported browser. It doesn't appear in the list of apps that have been found and a rescan has no effect. It was working successfully until the last update of Browsersaurus. 

This seems to be similar to issues #156 and #563. The work arounds in those posts didn't work.  I also noticed this turned up in issue #576. 

- Chrome turns up in spotlight (however I have tried turning Applications on and off in the spotlight search)
- Confirm Chrome is installed in the /Applications folder. 
- I only have one instance of Chrome installed
- I have tried resetting Browsersaurus and also tried removing and reinstalling Browersaurus, no difference. 
- Browsersaurus does find all my other relevant apps and launches them with no issues. 

I am using:
- Browsersaurus 20.0.2, 
- Chrome Version 109.0.5414.87, on 
- MacOS 13.0.1.

Any clues, how do I get Browsersaurus to find Chrome?

Thanks for your help!

**Comment by will-stone** [comment]
Where is your Chrome app and what is it called? If it’s anything other than `/Applications/Google Chrome` then it won’t be found. B now looks up browsers by name in the Applications folder. This obviously has the drawback of requiring consistent, unmodified installation, but it means not having to rely on Spotlight any more.

**Comment by newviews** [comment]
Thanks so much! Problem solved. 

My Chrome app was called `/Applications/Chrome`. Updated to `/Applications/Google Chrome` and problem solved.

**Comment by vkravets** [comment]
I had installed Google Chrome at user Application folder after creating symlink at /Application B detects Google Chrome
